region/lowernorfair/west/Golden Torizo's Room.json

```diff
@@ -550,6 +939,69 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [2, 7],
+      "name": "G-Mode",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"heatFrames": 0}
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 7],
+      "name": "G-Mode, GT Kill, Remain in G-Mode",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"notable": "Safe Spot Kill"},
+        {"or": [
+          {"resourceAtMost": [{"type": "Missile", "count": 31}]},
+          "canInsaneJump"
+        ]},
+        {"or": [
+          {"ammo": {"type": "Super", "count": 30}},
+          {"and": [
+            "Charge",
+            {"or": [
+              "canBeVeryPatient",
+              {"and": [
+                "canBePatient",
+                "Wave"
+              ]},
+              {"and": [
+                "canBePatient",
+                "Spazer"
+              ]},
+              "Plasma",
+              {"and": [
+                "canBePatient",
+                {"ammo": {"type": "Super", "count": 15}}
+              ]}
+            ]}
+          ]}
+        ]},
+        {"heatFrames": 0}
+      ],
+      "setsFlags": ["f_DefeatedGoldenTorizo"],
+      "flashSuitChecked": true,
+      "note": [
+        "Start the fight with 31 Missiles or fewer in order to prevent GT from using its laser attack, which will enable Samus to stand next to it.",
+        "Having GT start with his Chozo Ball attack also will make it so there are drops which can be collected at the end of the fight, otherwise they will have been overloaded.",
```

Comment:
You probably don't get drops with this strat.

Reply:
When i was doing it, I did get drops in their spawn location, but only if that was his first attack